frappe/email/doctype/email_queue/email_queue.json

```diff
@@ -58,7 +58,8 @@
    "in_list_view": 1,
    "in_standard_filter": 1,
    "label": "Status",
-   "options": "\nNot Sent\nSending\nSent\nError\nExpired"
+   "options": "\nNot Sent\nSending\nSent\nError\nExpired",
+   "search_index": 1
```

Comment:
There's already a multi-column index where status is the outermost column. 

https://github.com/frappe/frappe/blob/26d0e7a0f93439cec52263411b5c7db482001996/frappe/email/doctype/email_queue/email_queue.py#L370-L372